vsts/pipelines/copySdksFromProdToStorageAccount.yml

```diff
@@ -37,10 +44,18 @@ jobs:
           DEV_STORAGE_SAS_TOKEN: $(DEV-STORAGE-SAS-TOKEN)
           SANDBOX_STORAGE_SAS_TOKEN: $(SANDBOX-STORAGE-SAS-TOKEN)
           PROD_STORAGE_SAS_TOKEN: $(PROD-STORAGE-SAS-TOKEN)
+          PROD_BACKUP_STORAGE_SAS_TOKEN: $(PROD-BACKUP-STORAGE-SAS-TOKEN)
           PERSONAL_STORAGE_SAS_TOKEN: $(${{ upper(parameters.destinationStorageAccountName) }}-PERSONAL-STORAGE-SAS-TOKEN)
         inputs:
           scriptPath: ./vsts/scripts/copySdksFromProdToStorageAccount.sh
-          args:  ${{ parameters.destinationStorageAccountName }} ${{ parameters.dryRun }}
+          args:  ${{ parameters.destinationStorageAccountName }} ${{ parameters.dryRun }} ${{ parameters.overwrite }}
 
 trigger: none
 pr: none
+schedules:
+- cron: "0 12 * * 0"
+  displayName: Weekly Sunday Copy to Backup Prod
+  branches:
+    include:
+    - main
+  always: true # whether to always run the pipeline even if there have been source code changes since the last run
```

Comment:
Kind of a bummer, but couldn't find a way to pass in parameters for this run, so needed to update the defaults of the job to be what we want to run on the schedule